### PR TITLE
HW3, work with data types and operators

### PR DESCRIPTION
**Pull request by NykoEugen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (acman) [inline .gitignore:1]: 👍

File: HW3/task_1_data_type.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("I love Python \n" * 42)
```
Comment (acman) [inline HW3/task_1_data_type.py]: style nit: use blank line after code in all files

File: HW3/task_2.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+age = 27
+my_name = "Eugen"
+age_in_month = age*12
```
Comment (acman) [inline HW3/task_2.py]: style nit: use spaces before and after the operation symbol for more readability
```suggestion
age_in_month = age * 12
```

File: HW3/task_2.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+age_in_years = int(age_in_month/12)
```
Comment (acman) [inline HW3/task_2.py]:
```suggestion
age_in_years = int(age_in_month / 12)
```

File: HW3/task_4.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+a = 2
+b = 5
+c = 6
+
+d = str(a)+str(b)+str(c)
```
Comment (acman) [inline HW3/task_4.py]:
```suggestion
d = str(a) + str(b) + str(c)
```

**Review by acman — APPROVED** [review]
Great job! Added several comments on how to improve the code but anyway I approved the PR

File: HW3/task_2.py (hunk 3 above)
Comment (NykoEugen) [reply HW3/task_2.py]: got it

**Pull request merged**